file_system.go

```diff
@@ -446,6 +449,9 @@ func parseMLST(entry string, skipSelfParent bool) (os.FileInfo, error) {
 
 	if typ == "dir" || typ == "cdir" || typ == "pdir" {
 		mode |= os.ModeDir
+	} else if strings.HasPrefix(typ, "os.unix=slink") || strings.HasPrefix(typ, "os.unix=symlink") {
```

Comment:
Is there a reason these can't be == checks? Is it the ":" that comes after slink?